src/prefect/task_runners.py

```diff
@@ -109,6 +109,21 @@ def concurrency_type(self) -> TaskConcurrencyType:
     def name(self):
         return type(self).__name__.lower().replace("taskrunner", "")
 
+    def duplicate(self):
+        """
+        Return a new task runner instance with the same options.
+        """
+        return type(self)()
+
+    def __eq__(self, __value: object) -> bool:
+        """
+        Returns true if the task runners use the same options.
+        """
+        if type(__value) == type(self):
+            return True
+        else:
+            return NotImplemented
```

Comment:
question: is the intention here to let the other type determine equality?

Reply:
Yep exactly this signals to Python that it should call __eq__ on the other type.